test-suite/djinni/single_language_interfaces.djinni

```diff
@@ -8,4 +9,6 @@ uses_single_language_listeners = interface +o +j +c {
   returnForObjC(): objc_only_listener;
   callForJava(l: java_only_listener);
   returnForJava(): java_only_listener;
+  callForPy(l: py_only_listener);
+  returnForPy(): py_only_listener;
```

Comment:
I haven't yet figured out how this test works or if it is executed in Python, but there might be a `+p` missing in the interface definition?

```
uses_single_language_listeners = interface +o +j +c +p {
```

Reply:
It's most likely not being executed in Python then :(